src/main/java/org/folio/clients/OkapiClients.java

```diff
@@ -0,0 +1,27 @@
+package org.folio.clients;
+
+import org.folio.clients.impl.SourceRecordStorageClient;
+import org.folio.util.OkapiConnectionParams;
+
+public class OkapiClients {
+
+  private final OkapiConnectionParams okapiConnectionParams;
+
+  private SynchronousOkapiClient sourceRecordStorageClient;
+
+
+  private OkapiClients(OkapiConnectionParams okapiConnectionParams) {
+    this.okapiConnectionParams = okapiConnectionParams;
+  }
+
+  public static OkapiClients create(OkapiConnectionParams okapiConnectionParams) {
+    return new OkapiClients(okapiConnectionParams);
+  }
+
+  public SynchronousOkapiClient getSourceRecordStorageClient() {
+    if (sourceRecordStorageClient == null) {
```

Comment:
This initialization is safe only in the single-thread application. We probably will use multiple worker-threads, sourceRecordStorageClient should be initialized safety